addons/sale/models/sale.py

```diff
@@ -1489,7 +1489,7 @@ def product_uom_change(self):
     def name_get(self):
         result = []
         for so_line in self.sudo():
-            name = '%s - %s' % (so_line.order_id.name, so_line.name.split('\n')[0] or so_line.product_id.name)
+            name = '%s - %s' % (so_line.order_id.name, so_line.name and so_line.name.split('\n')[0] or so_line.product_id.name)
```

Comment:
Hi @wtaferner I ave one remark :
- I think it is better to use `display_name` indeed `name` of product `or so_line.product_id.name` ->` or so_line.product_id.display_name`

Reply:
@fmdl 
It is a bugfix, so it would be the call of @nim-odoo, but it makes sense somehow what you suggest.